standard/basic-concepts.md

```diff
@@ -668,6 +680,7 @@ Name hiding through nesting can occur as a result of nesting namespaces or types
 
 > *Example*: In the following code
 >
+> <!-- Example: {template:"standalone-lib",name:"HidingNesting1", ignoredWarnings:["CS0219"],ignoredWarnings:["CS0414"]} -->
```

Comment:
I suspect second of these `ignoredWarnings` will override (replace) the first. If you want to ignore multiple warnings, specify them in the array.

```suggestion
> <!-- Example: {template:"standalone-lib",name:"HidingNesting1", ignoredWarnings:["CS0219", "CS0414"]} -->
```